src/app/index.config.coffee

```diff
@@ -60,3 +60,7 @@
     '''
     })
   )
+  # https://stackoverflow.com/questions/41063947/angular-1-6-0-possibly-unhandled-rejection-error
+  .config(($qProvider) ->
+    $qProvider.errorOnUnhandledRejections(false)
```

Comment:
This workaround only masks the errors...

(cf. https://stackoverflow.com/questions/41281515/possibly-unhandled-rejection-in-angular-1-6/41283198#41283198)